pkg/build/api/types.go

```diff
@@ -559,10 +556,30 @@ type BuildConfigSpec struct {
 	// are defined, a new build can only occur as a result of an explicit client build creation.
 	Triggers []BuildTriggerPolicy
 
+	// SchedulePolicy describes how the new build created from this build
+	// configuration will be scheduled for execution.
+	// This is optional, if not specified we default to "parallel".
+	SchedulePolicy *BuildSchedulePolicy
```

Comment:
@smarterclayton API question... do you think this field should be here or in BuildSpec? I think it should be higher level (same as Triggers), WDYT?